src/main/java/de/adorsys/keycloak/config/service/ClientImportService.java

```diff
@@ -67,20 +73,18 @@ public void doImport(RealmImport realmImport) {
             return;
         }
 
-        createOrUpdateClients(realmImport, clients);
+        createOrUpdateClients(realmImport, clients.stream()
```

Comment:
I think using deepClone here is a bit overkill since deepClone feels like a cost intensive operation.

authenticationFlowBindingOverrides could follow the same way like authorizationSettings. You could the that authorizationSettings is already mention on some DeepUtils calls (line 116, 127, 132) or does this not work?

Reply:
I miss this. I'll fix that